tokio/src/fs/mod.rs

```diff
@@ -18,12 +18,10 @@
 //! type. Adaptions also extend to traits like `std::io::Read` where methods
 //! return `std::io::Result`.  Be warned that these adapted methods may return
 //! `std::io::ErrorKind::WouldBlock` if a *worker* thread can not be converted
-//! to a *backup* thread immediately. See [tokio-executor] for more details
-//! of the threading model and [`blocking`].
+//! to a *backup* thread immediately.
 //!
-//! [`blocking`]: https://docs.rs/tokio-executor/0.2.0-alpha.2/tokio_executor/threadpool/fn.blocking.html
+//! [`blocking`]: #
```

Comment:
missing link?

Reply:
There is nothing to link to until #1681 lands. I opted to leave the comment vs. deleting it.

Reply:
I would just not _manually_ give the link, and have the rust auto-doc stuff take care of it. It will leave a warning if the symbol doesn't resolve. If it's left like this, we _will_ forget to change it.

Reply:
Done.